AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/Model/KeyStore.smithy

```diff
@@ -102,7 +102,8 @@ structure KeyStoreConfig {
 }
 
 union KMSConfiguration {
-  kmsKeyArn: com.amazonaws.kms#KeyIdType
+  kmsKeyArn: com.amazonaws.kms#KeyIdType,
+  mrkKmsKeyArn: com.amazonaws.kms#KeyIdType
```

Comment:
Blocking (or should have been): MRK means Multi-Region Key.
This line translates to: Multi-Region Key Key Management Service Key Amazon Resource.

I am going to replace this with: `kmsMRKeyArn`